src/authInfo.ts

```diff
@@ -229,7 +229,7 @@ function getJwtAudienceUrl(options: OAuth2Options & { createdOrgInstance?: strin
     createdOrgInstance.endsWith('s') ||
     loginUrlLowercased?.includes('sandbox.my.salesforce.com') || // enhanced domains >= 230
     loginUrlLowercased?.match(/(cs[0-9]+(\.my|)\.salesforce\.com)/g) || // my domains on CS instance OR CS instance without my domain
-    loginUrlLowercased?.match(/(usa[0-9]+s\..+\.salesforce\.com)/g) || // falcon sandbox ex: usa2s.sfdc-whatever.salesforce.com
+    loginUrlLowercased?.match(/((usa|ind|aus)[0-9]+s\..+\.salesforce\.com)/g) || // falcon sandbox ex: usa2s.sfdc-whatever.salesforce.com
```

Comment:
@mshanemc specifying region names is fragile. This will need to be adjusted over time.

Reply:
What's a good way to not do it?  How do I keep a prod url for some hipster vinyl business like vintage78s.my.salesforce.com from being treated as a sandbox because of falcon?

Reply:
That's a good point, but there has to be a way to distinguish a my domain URL from an instance login URL.

Reply:
I agree, this is going to be a huge pain to update this list every time a new falcon instance is added.

I would rather to take our chances with the hipsters, or even something like `/([a-z]{3}[0-9]+s\..+\.salesforce\.com)/gi)` Note the `i` for case insenstivity.

Reply:
Pete had more patterns over in slack (sfdc-something) that may help.  Stand by for another push.